lib/std/process.zig

```diff
@@ -1746,7 +1746,12 @@ pub const TotalSystemMemoryError = error{
 pub fn totalSystemMemory() TotalSystemMemoryError!u64 {
     switch (native_os) {
         .linux => {
-            return totalSystemMemoryLinux() catch return error.UnknownTotalSystemMemory;
```

Comment:
Probably no reason to leave `totalSystemMemoryLinux` around since it's a private function.